Index: frontend/.gitignore
===================================================================
diff --git a/frontend/.gitignore b/frontend/.gitignore
deleted file mode 100644
--- a/frontend/.gitignore	(revision 7d17fffc0510bd8304f68d96f233f1d94d0fe0ea)
+++ /dev/null	(revision 7d17fffc0510bd8304f68d96f233f1d94d0fe0ea)
@@ -1,23 +0,0 @@
-.DS_Store
-node_modules
-/dist
-
-
-# local env files
-.env.local
-.env.*.local
-
-# Log files
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
-pnpm-debug.log*
-
-# Editor directories and files
-.idea
-.vscode
-*.suo
-*.ntvs*
-*.njsproj
-*.sln
-*.sw?
Index: src/main/java/com/tasty/reviews/tastyreviews/dto/MemberDTO.java
===================================================================
diff --git a/src/main/java/com/tasty/reviews/tastyreviews/dto/MemberDTO.java b/src/main/java/com/tasty/reviews/tastyreviews/dto/MemberDTO.java
deleted file mode 100644
--- a/src/main/java/com/tasty/reviews/tastyreviews/dto/MemberDTO.java	(revision 7d17fffc0510bd8304f68d96f233f1d94d0fe0ea)
+++ /dev/null	(revision 7d17fffc0510bd8304f68d96f233f1d94d0fe0ea)
@@ -1,44 +0,0 @@
-//package com.tasty.reviews.tastyreviews.dto;
-//
-//import com.tasty.reviews.tastyreviews.domain.Gender;
-//import com.tasty.reviews.tastyreviews.domain.Member;
-//import lombok.AllArgsConstructor;
-//import lombok.Builder;
-//import lombok.Data;
-//import lombok.NoArgsConstructor;
-//
-//@Data
-//@Builder
-//@AllArgsConstructor
-//@NoArgsConstructor
-//public class MemberDTO { //회원가입에서 클라이언트가 보낸 정보를 전달하는 DTO
-//
-//    private Long id;
-//    private String email;
-//    private String password;
-//    private int age;
-//    private String nickname;
-//    private Gender gender;
-//
-//    public Member toEntity() {
-//        return Member.builder()
-//                .id(this.id)
-//                .email(this.email)
-//                .password(this.password)
-//                .age(this.age)
-//                .nickname(this.nickname)
-//                .gender(this.gender)
-//                .build();
-//    }
-//
-//    public static MemberDTO fromEntity(Member member) {
-//        return MemberDTO.builder()
-//                .id(member.getId())
-//                .email(member.getEmail())
-//                .password(member.getPassword())
-//                .age(member.getAge())
-//                .nickname(member.getNickname())
-//                .gender(member.getGender())
-//                .build();
-//    }
-//}
\ No newline at end of file
Index: frontend/README.md
===================================================================
diff --git a/frontend/README.md b/frontend/README.md
deleted file mode 100644
--- a/frontend/README.md	(revision 7d17fffc0510bd8304f68d96f233f1d94d0fe0ea)
+++ /dev/null	(revision 7d17fffc0510bd8304f68d96f233f1d94d0fe0ea)
@@ -1,24 +0,0 @@
-# frontend
-
-## Project setup
-```
-npm install
-```
-
-### Compiles and hot-reloads for development
-```
-npm run serve
-```
-
-### Compiles and minifies for production
-```
-npm run build
-```
-
-### Lints and fixes files
-```
-npm run lint
-```
-
-### Customize configuration
-See [Configuration Reference](https://cli.vuejs.org/config/).
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<project version=\"4\">\n  <component name=\"ProjectModuleManager\">\n    <modules>\n      <module fileurl=\"file://$PROJECT_DIR$/.idea/modules/com.tasty.reviews.tasty-reviews.main.iml\" filepath=\"$PROJECT_DIR$/.idea/modules/com.tasty.reviews.tasty-reviews.main.iml\" />\n      <module fileurl=\"file://$PROJECT_DIR$/.idea/modules/tasty-reviews.iml\" filepath=\"$PROJECT_DIR$/.idea/modules/tasty-reviews.iml\" />\n      <module fileurl=\"file://$PROJECT_DIR$/.idea/modules/tasty-reviews.test.iml\" filepath=\"$PROJECT_DIR$/.idea/modules/tasty-reviews.test.iml\" />\n    </modules>\n  </component>\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
--- a/.idea/modules.xml	(revision 7d17fffc0510bd8304f68d96f233f1d94d0fe0ea)
+++ b/.idea/modules.xml	(date 1713779659727)
@@ -2,9 +2,7 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
-      <module fileurl="file://$PROJECT_DIR$/.idea/modules/com.tasty.reviews.tasty-reviews.main.iml" filepath="$PROJECT_DIR$/.idea/modules/com.tasty.reviews.tasty-reviews.main.iml" />
-      <module fileurl="file://$PROJECT_DIR$/.idea/modules/tasty-reviews.iml" filepath="$PROJECT_DIR$/.idea/modules/tasty-reviews.iml" />
-      <module fileurl="file://$PROJECT_DIR$/.idea/modules/tasty-reviews.test.iml" filepath="$PROJECT_DIR$/.idea/modules/tasty-reviews.test.iml" />
+      <module fileurl="file://$PROJECT_DIR$/.idea/modules/tasty-reviews.main.iml" filepath="$PROJECT_DIR$/.idea/modules/tasty-reviews.main.iml" />
     </modules>
   </component>
 </project>
\ No newline at end of file
Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>### IntelliJ IDEA ###\nout/\n!**/src/main/**/out/\n!**/src/test/**/out/\n\n### Eclipse ###\n.apt_generated\n.classpath\n.factorypath\n.project\n.settings\n.springBeans\n.sts4-cache\nbin/\n!**/src/main/**/bin/\n!**/src/test/**/bin/\n\n### NetBeans ###\n/nbproject/private/\n/nbbuild/\n/dist/\n/nbdist/\n/.nb-gradle/\n\n### VS Code ###\n.vscode/\n\n### Mac OS ###\n.DS_Store\n# 프로젝트 제외 경로\n/.gradle/
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 7d17fffc0510bd8304f68d96f233f1d94d0fe0ea)
+++ b/.gitignore	(date 1713779659735)
@@ -28,4 +28,147 @@
 ### Mac OS ###
 .DS_Store
 # 프로젝트 제외 경로
-/.gradle/
\ No newline at end of file
+/.gradle/
+/.idea/sonarlint/issuestore/0/d/0d16f03dc0b15d4646efba5e51f694c283029e20
+/.idea/sonarlint/securityhotspotstore/0/d/0d16f03dc0b15d4646efba5e51f694c283029e20
+/.idea/sonarlint/issuestore/1/a/1a273f73b7a477b0745593641bd708a6182c7ad5
+/.idea/sonarlint/securityhotspotstore/1/a/1a273f73b7a477b0745593641bd708a6182c7ad5
+/.idea/sonarlint/issuestore/2/a/2a45a911a8f1836b0b6c5b758962572012d8f8c3
+/.idea/sonarlint/securityhotspotstore/2/a/2a45a911a8f1836b0b6c5b758962572012d8f8c3
+/.idea/sonarlint/issuestore/4/b/4b4b6b66529bc3a9a37aab50e6d661a9782be256
+/.idea/sonarlint/securityhotspotstore/4/b/4b4b6b66529bc3a9a37aab50e6d661a9782be256
+/.idea/sonarlint/issuestore/4/b/4baf1ce38226df7b01754e4b8a71e3c205c0bd7b
+/.idea/sonarlint/securityhotspotstore/4/b/4baf1ce38226df7b01754e4b8a71e3c205c0bd7b
+/.idea/sonarlint/issuestore/4/b/4bfa5919ef29d5ed2581aa1a6a3a64baa87abb89
+/.idea/sonarlint/securityhotspotstore/4/b/4bfa5919ef29d5ed2581aa1a6a3a64baa87abb89
+/.idea/sonarlint/issuestore/4/e/4e4dc9ee428b2605ae39dad2a4439c0b6c2a93d3
+/.idea/sonarlint/securityhotspotstore/4/e/4e4dc9ee428b2605ae39dad2a4439c0b6c2a93d3
+/.idea/sonarlint/issuestore/5/b/5bbfa66edb4db3c7c33c5181f43510990d3307f9
+/.idea/sonarlint/securityhotspotstore/5/b/5bbfa66edb4db3c7c33c5181f43510990d3307f9
+/.idea/sonarlint/issuestore/0/5/05efc8b1657769a27696d478ded1e95f38737233
+/.idea/sonarlint/securityhotspotstore/0/5/05efc8b1657769a27696d478ded1e95f38737233
+/.idea/sonarlint/issuestore/6/f/6f7505a483624fd190218f37ed52b93fb4df2136
+/.idea/sonarlint/securityhotspotstore/6/f/6f7505a483624fd190218f37ed52b93fb4df2136
+/.idea/sonarlint/issuestore/9/c/9ca1036eaf2122c493f9fa04ea610d0173d0d76a
+/.idea/sonarlint/securityhotspotstore/9/c/9ca1036eaf2122c493f9fa04ea610d0173d0d76a
+/.idea/sonarlint/issuestore/0/9/09de52530696a3c3ffc55528ead70f5a37258955
+/.idea/sonarlint/securityhotspotstore/0/9/09de52530696a3c3ffc55528ead70f5a37258955
+/.idea/sonarlint/issuestore/1/8/18d1eb850eef616a8b07d7601b86fbfb9e157caa
+/.idea/sonarlint/securityhotspotstore/1/8/18d1eb850eef616a8b07d7601b86fbfb9e157caa
+/.idea/sonarlint/issuestore/6/9/69dab7641aba081f8415c636eb5a5eb3a5bf7c9a
+/.idea/sonarlint/securityhotspotstore/6/9/69dab7641aba081f8415c636eb5a5eb3a5bf7c9a
+/.idea/sonarlint/issuestore/4/2/420fc7e3ab5a1a191f5cb6c6e51db6f436d57d08
+/.idea/sonarlint/securityhotspotstore/4/2/420fc7e3ab5a1a191f5cb6c6e51db6f436d57d08
+/.idea/sonarlint/issuestore/7/3/733a7f78fb15f866556fb34f314e41cbd53ce05f
+/.idea/sonarlint/securityhotspotstore/7/3/733a7f78fb15f866556fb34f314e41cbd53ce05f
+/.idea/sonarlint/issuestore/3/9/3967f29d78377b763163e88658a43b154fb3d247
+/.idea/sonarlint/securityhotspotstore/3/9/3967f29d78377b763163e88658a43b154fb3d247
+/.idea/sonarlint/issuestore/0/4/04756c7b7e8e43f17169deb01888be2d3b270d8b
+/.idea/sonarlint/securityhotspotstore/0/4/04756c7b7e8e43f17169deb01888be2d3b270d8b
+/.idea/sonarlint/issuestore/4/8/4821e449ac548ef36b5fc3ffdc4eba73e6d42e0a
+/.idea/sonarlint/securityhotspotstore/4/8/4821e449ac548ef36b5fc3ffdc4eba73e6d42e0a
+/.idea/sonarlint/issuestore/4/8/4890cd8a5d9aef815f862427dfcb01d52d451fac
+/.idea/sonarlint/securityhotspotstore/4/8/4890cd8a5d9aef815f862427dfcb01d52d451fac
+/.idea/sonarlint/issuestore/6/1/61749c0b3364dc867ad751ebc519407e4196e7a3
+/.idea/sonarlint/securityhotspotstore/6/1/61749c0b3364dc867ad751ebc519407e4196e7a3
+/.idea/sonarlint/issuestore/4/4/442428b3cdad8762cbb3d0ac69c7df0ea1e37c23
+/.idea/sonarlint/securityhotspotstore/4/4/442428b3cdad8762cbb3d0ac69c7df0ea1e37c23
+/.idea/sonarlint/issuestore/a/5/a5cc2925ca8258af241be7e5b0381edf30266302
+/.idea/sonarlint/securityhotspotstore/a/5/a5cc2925ca8258af241be7e5b0381edf30266302
+/.idea/sonarlint/issuestore/a/9/a951e9c1b74a198fec6827507856d3995acb2822
+/.idea/sonarlint/securityhotspotstore/a/9/a951e9c1b74a198fec6827507856d3995acb2822
+/build/classes/java/main/com/tasty/reviews/tastyreviews/controller/AdminController.class
+/.idea/sonarlint/issuestore/a/f/af71118b1c5790e1c5fa49b764dc744a2b04bbbf
+/.idea/sonarlint/securityhotspotstore/a/f/af71118b1c5790e1c5fa49b764dc744a2b04bbbf
+/build/resources/main/application.yml
+/build/resources/test/application.yml
+/build/reports/tests/test/css/base-style.css
+/build/classes/java/main/com/tasty/reviews/tastyreviews/domain/BaseTimeEntity.class
+/.idea/sonarlint/issuestore/c/3/c376ee403e0d874cf383d4c38717ee6f20faa392
+/.idea/sonarlint/securityhotspotstore/c/3/c376ee403e0d874cf383d4c38717ee6f20faa392
+/build/reports/tests/test/packages/com.tasty.reviews.tastyreviews.controller.html
+/build/reports/tests/test/classes/com.tasty.reviews.tastyreviews.controller.MemberControllerTest.html
+/build/classes/java/main/com/tasty/reviews/tastyreviews/config/CorsMvcConfig.class
+/build/classes/java/main/com/tasty/reviews/tastyreviews/dto/CreateMemberDTO$CreateMemberDTOBuilder.class
+/build/classes/java/main/com/tasty/reviews/tastyreviews/dto/CreateMemberDTO.class
+/build/classes/java/main/com/tasty/reviews/tastyreviews/dto/CustomUserDetails.class
+/build/classes/java/main/com/tasty/reviews/tastyreviews/service/CustomUserDetailsService.class
+/.idea/sonarlint/issuestore/d/3/d3b4887ab3557c3a265017a75791661787dc6e7d
+/.idea/sonarlint/securityhotspotstore/d/3/d3b4887ab3557c3a265017a75791661787dc6e7d
+/.idea/sonarlint/issuestore/d/9/d912f772af6813af6987854a77cafd5fdb694201
+/.idea/sonarlint/securityhotspotstore/d/9/d912f772af6813af6987854a77cafd5fdb694201
+/.idea/sonarlint/issuestore/d/a/daa7bbb759af16c1ab8665758bf13c6ad1188154
+/.idea/sonarlint/securityhotspotstore/d/a/daa7bbb759af16c1ab8665758bf13c6ad1188154
+/.idea/sonarlint/issuestore/e/2/e2e2d3ed4ce4b4edd74d4d490f75269aa4b28dcd
+/.idea/sonarlint/securityhotspotstore/e/2/e2e2d3ed4ce4b4edd74d4d490f75269aa4b28dcd
+/.idea/sonarlint/issuestore/e/a/ea262b8a246cbbeefd6c20e3e9bd608f394bd67b
+/.idea/sonarlint/securityhotspotstore/e/a/ea262b8a246cbbeefd6c20e3e9bd608f394bd67b
+/build/classes/java/main/com/tasty/reviews/tastyreviews/exception/ExceptionManager.class
+/.idea/sonarlint/issuestore/f/6/f6967ad820d9688aeef36050621da1c8b54baf1f
+/.idea/sonarlint/securityhotspotstore/f/6/f6967ad820d9688aeef36050621da1c8b54baf1f
+/.idea/sonarlint/issuestore/f/0/f07866736216be0ee2aba49e392191aeae700a35
+/.idea/sonarlint/securityhotspotstore/f/0/f07866736216be0ee2aba49e392191aeae700a35
+/.idea/sonarlint/issuestore/f/b/fbe448ebfc3eb2d4e308f6b8b043666f5b57235e
+/.idea/sonarlint/securityhotspotstore/f/b/fbe448ebfc3eb2d4e308f6b8b043666f5b57235e
+/build/classes/java/main/com/tasty/reviews/tastyreviews/domain/Gender.class
+/HELP.md
+/build/reports/tests/test/index.html
+/.idea/sonarlint/issuestore/index.pb
+/.idea/sonarlint/securityhotspotstore/index.pb
+/.idea/jpa.xml
+/.idea/jpa-buddy.xml
+/.idea/jsLibraryMappings.xml
+/build/classes/java/main/com/tasty/reviews/tastyreviews/jwt/JWTFilter.class
+/build/classes/java/main/com/tasty/reviews/tastyreviews/jwt/JWTUtil.class
+/build/classes/java/main/com/tasty/reviews/tastyreviews/jwt/LoginFilter.class
+/build/tmp/compileTestJava/compileTransaction/stash-dir/LoginMemberServiceTest.class.uniqueId1
+/build/classes/java/main/com/tasty/reviews/tastyreviews/controller/MainController.class
+/build/resources/main/map.html
+/build/classes/java/main/com/tasty/reviews/tastyreviews/domain/Member$MemberBuilder.class
+/build/classes/java/main/com/tasty/reviews/tastyreviews/domain/Member.class
+/build/classes/java/main/com/tasty/reviews/tastyreviews/controller/MemberController.class
+/build/tmp/compileTestJava/compileTransaction/stash-dir/MemberControllerTest.class.uniqueId0
+/build/classes/java/main/com/tasty/reviews/tastyreviews/repository/MemberRepository.class
+/build/classes/java/main/com/tasty/reviews/tastyreviews/service/MemberService.class
+/build/test-results/test/binary/output.bin
+/build/test-results/test/binary/output.bin.idx
+/build/tmp/compileJava/previous-compilation-data.bin
+/build/tmp/compileTestJava/previous-compilation-data.bin
+/.idea/inspectionProfiles/Project_Default.xml
+/build/reports/tests/test/js/report.js
+/build/classes/java/main/com/tasty/reviews/tastyreviews/domain/Restaurant$RestaurantBuilder.class
+/build/tmp/compileJava/compileTransaction/stash-dir/Restaurant$RestaurantBuilder.class.uniqueId11
+/build/classes/java/main/com/tasty/reviews/tastyreviews/domain/Restaurant.class
+/build/tmp/compileJava/compileTransaction/stash-dir/Restaurant.class.uniqueId12
+/build/classes/java/main/com/tasty/reviews/tastyreviews/controller/RestaurantController.class
+/build/tmp/compileJava/compileTransaction/stash-dir/RestaurantController.class.uniqueId9
+/build/classes/java/main/com/tasty/reviews/tastyreviews/dto/RestaurantDTO$RestaurantDTOBuilder.class
+/build/tmp/compileJava/compileTransaction/stash-dir/RestaurantDTO$RestaurantDTOBuilder.class.uniqueId0
+/build/classes/java/main/com/tasty/reviews/tastyreviews/dto/RestaurantDTO.class
+/build/tmp/compileJava/compileTransaction/stash-dir/RestaurantDTO.class.uniqueId13
+/build/classes/java/main/com/tasty/reviews/tastyreviews/repository/RestaurantRepository.class
+/build/tmp/compileJava/compileTransaction/stash-dir/RestaurantRepository.class.uniqueId4
+/build/classes/java/main/com/tasty/reviews/tastyreviews/service/RestaurantService.class
+/build/tmp/compileJava/compileTransaction/stash-dir/RestaurantService.class.uniqueId7
+/build/test-results/test/binary/results.bin
+/build/classes/java/main/com/tasty/reviews/tastyreviews/domain/Review$ReviewBuilder.class
+/build/tmp/compileJava/compileTransaction/stash-dir/Review$ReviewBuilder.class.uniqueId3
+/build/classes/java/main/com/tasty/reviews/tastyreviews/domain/Review.class
+/build/tmp/compileJava/compileTransaction/stash-dir/Review.class.uniqueId1
+/build/classes/java/main/com/tasty/reviews/tastyreviews/controller/ReviewController.class
+/build/tmp/compileJava/compileTransaction/stash-dir/ReviewController.class.uniqueId6
+/build/classes/java/main/com/tasty/reviews/tastyreviews/dto/ReviewDTO$ReviewDTOBuilder.class
+/build/tmp/compileJava/compileTransaction/stash-dir/ReviewDTO$ReviewDTOBuilder.class.uniqueId10
+/build/classes/java/main/com/tasty/reviews/tastyreviews/dto/ReviewDTO.class
+/build/tmp/compileJava/compileTransaction/stash-dir/ReviewDTO.class.uniqueId8
+/build/classes/java/main/com/tasty/reviews/tastyreviews/repository/ReviewRepository.class
+/build/tmp/compileJava/compileTransaction/stash-dir/ReviewRepository.class.uniqueId5
+/build/classes/java/main/com/tasty/reviews/tastyreviews/service/ReviewService.class
+/build/tmp/compileJava/compileTransaction/stash-dir/ReviewService.class.uniqueId2
+/build/classes/java/main/com/tasty/reviews/tastyreviews/domain/Role.class
+/build/classes/java/main/com/tasty/reviews/tastyreviews/config/SecurityConfig.class
+/build/reports/tests/test/css/style.css
+/build/classes/java/main/com/tasty/reviews/tastyreviews/TastyReviewsApplication.class
+/build/classes/java/test/com/tasty/reviews/tastyreviews/TastyReviewsApplicationTests.class
+/build/test-results/test/TEST-com.tasty.reviews.tastyreviews.controller.MemberControllerTest.xml
+/build/classes/java/main/com/tasty/reviews/tastyreviews/controller/TestController.class
Index: .idea/compiler.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<project version=\"4\">\n  <component name=\"CompilerConfiguration\">\n    <annotationProcessing>\n      <profile default=\"true\" name=\"Default\" enabled=\"true\" />\n      <profile name=\"Gradle Imported\" enabled=\"true\">\n        <outputRelativeToContentRoot value=\"true\" />\n        <processorPath useClasspath=\"false\">\n          <entry name=\"$USER_HOME$/.gradle/caches/modules-2/files-2.1/org.projectlombok/lombok/1.18.30/f195ee86e6c896ea47a1d39defbe20eb59cd149d/lombok-1.18.30.jar\" />\n        </processorPath>\n        <module name=\"com.tasty.reviews.tasty-reviews.main\" />\n      </profile>\n    </annotationProcessing>\n    <bytecodeTargetLevel target=\"17\" />\n  </component>\n  <component name=\"JavacSettings\">\n    <option name=\"ADDITIONAL_OPTIONS_STRING\" value=\"-parameters\" />\n  </component>\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/compiler.xml b/.idea/compiler.xml
--- a/.idea/compiler.xml	(revision 7d17fffc0510bd8304f68d96f233f1d94d0fe0ea)
+++ b/.idea/compiler.xml	(date 1713779659744)
@@ -8,7 +8,7 @@
         <processorPath useClasspath="false">
           <entry name="$USER_HOME$/.gradle/caches/modules-2/files-2.1/org.projectlombok/lombok/1.18.30/f195ee86e6c896ea47a1d39defbe20eb59cd149d/lombok-1.18.30.jar" />
         </processorPath>
-        <module name="com.tasty.reviews.tasty-reviews.main" />
+        <module name="tasty-reviews.main" />
       </profile>
     </annotationProcessing>
     <bytecodeTargetLevel target="17" />
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<project version=\"4\">\n  <component name=\"ExternalStorageConfigurationManager\" enabled=\"true\" />\n  <component name=\"FrameworkDetectionExcludesConfiguration\">\n    <file type=\"web\" url=\"file://$PROJECT_DIR$\" />\n  </component>\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_17\" default=\"true\" project-jdk-name=\"17\" project-jdk-type=\"JavaSDK\" />\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision 7d17fffc0510bd8304f68d96f233f1d94d0fe0ea)
+++ b/.idea/misc.xml	(date 1713779659747)
@@ -5,4 +5,7 @@
     <file type="web" url="file://$PROJECT_DIR$" />
   </component>
   <component name="ProjectRootManager" version="2" languageLevel="JDK_17" default="true" project-jdk-name="17" project-jdk-type="JavaSDK" />
+  <component name="ProjectType">
+    <option name="id" value="jpab" />
+  </component>
 </project>
\ No newline at end of file
